installer.sh.in

```diff
@@ -809,22 +826,30 @@ set_bootloader() {
     chroot $TARGETDIR grub-install $grub_args $dev >$LOG 2>&1
     if [ $? -ne 0 ]; then
         DIALOG --msgbox "${BOLD}${RED}ERROR:${RESET} \
-        failed to install GRUB to $dev!\nCheck $LOG for errors." ${MSGBOXSIZE}
+failed to install GRUB to $dev!\nCheck $LOG for errors." ${MSGBOXSIZE}
         DIE 1
     fi
     echo "Running grub-mkconfig on $TARGETDIR..." >$LOG
     chroot $TARGETDIR grub-mkconfig -o /boot/grub/grub.cfg >$LOG 2>&1
     if [ $? -ne 0 ]; then
         DIALOG --msgbox "${BOLD}${RED}ERROR${RESET}: \
-        failed to run grub-mkconfig!\nCheck $LOG for errors." ${MSGBOXSIZE}
+failed to run grub-mkconfig!\nCheck $LOG for errors." ${MSGBOXSIZE}
         DIE 1
     fi
 }
 
 test_network() {
-    rm -f otime && \
-        xbps-uhelper fetch https://repo-default.voidlinux.org/current/otime >$LOG 2>&1
-    if [ $? -eq 0 ]; then
+    # Pick a peace of data to download to ensure that a network connection is
+    # available.
+    local file="https://repo-default.voidlinux.org/current/otime"
+
+    # Download file with XBPS, remove it directly afterwards
+    xbps-uhelper fetch "$file" >$LOG 2>&1
+    local status=$?
+    rm -f xtraeme.asc
```

Comment:
```suggestion
    rm -f otime
```

Reply:
Thanks a lot! I've adjusted the file name.